tests/integration/replay/test_output.good

```diff
@@ -0,0 +1,46 @@
+makeMemoryResource HOST
+makeAllocator <umpire::strategy::DynamicPool>(host_simpool_defaults, getAllocator(HOST))
+makeAllocator <umpire::strategy::DynamicPool>(host_simpool_spec1, getAllocator(HOST), 9876, 1234)
+makeAllocator <umpire::strategy::DynamicPool>(host_simpool_spec2, getAllocator(HOST), 9876, 1234)
+makeAllocator <umpire::strategy::MonotonicAllocationStrategy>(MONOTONIC 1024, 1024, getAllocator(HOST))
+makeAllocator <umpire::strategy::SlotPool>(host_slot_pool, 64, getAllocator(HOST))
+makeAllocator <umpire::strategy::ThreadSafeAllocator>(thread_safe_allocator, getAllocator(HOST))
+allocate (17) host_simpool_defaults --> 1
```

Comment:
How do we get the names of Allocators?

Reply:
Or were we already recording it?

Reply:
The names were recorded at makeAllocator time.  Replay established a mapping of the name to the object reference.  It later looks for the object reference of other allocations and then performs the allocation by getting the allocator for the mapped name.